manifests/config.pp

```diff
@@ -58,6 +59,9 @@
   if $https_ports {
     create_resources('squid::https_port', $https_ports)
   }
+  if $snmp_listen_ip {
```

Comment:
If it's now a hash, I think `snmp_listen_ips` (plural) would be better and consistent with the other hash parameters.

Reply:
i changed it.
Could you also take a look on to my other Requests? (all with succeeded tests)